frontend/src/scenes/retention/RetentionTable.scss

```diff
@@ -42,10 +42,13 @@
         text-align: right;
         background-color: var(--retention-table-color); // NOTE: This is usually overridden in JS
         border-radius: var(--radius);
-        transition: transform 200ms ease-out;
 
-        &:hover {
-            transform: scale(1.05);
+        &.RetentionTable__Tab--clickable {
+            transition: transform 200ms ease-out;
+
+            &:hover {
+                transform: scale(1.05);
+            }
```

Comment:
Not sure if applicable here due to way it's all built, but we try to avoid using pure CSS files/rules whenever possible, and prefer tailwind classes.

Reply:
ya totally - not going to refactor this now since this was built this way, but good note